vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -483,6 +495,7 @@ unit:AT-PER-M
 unit:ATM
   a qudt:Unit ;
   dcterms:description "The standard atmosphere (symbol: atm) is an international reference pressure defined as \\(101.325 \\,kPa\\) and formerly used as unit of pressure. For practical purposes it has been replaced by the bar which is \\(100 kPa\\). The difference of about 1% is not significant for many applications, and is within the error range of common pressure gauges."^^qudt:LatexString ;
+  qudt:applicableUnit quantitykind:VaporPressure ;
```

Comment:
This is the wrong relation. A Unit does not have an applicableUnit. A Unit can have a qudt:hasQuantityKind, which is probably what you intended here, yes?

Reply:
(There are many of these errors in the units file)